docs/guides/manage-identities/20_create-identities.mdx

```diff
@@ -1,22 +1,10 @@
 ---
-id: managing-users-identities
-title: Create and invite identities
+id: create-identities
+title: Create Identities
```

Comment:
```suggestion
title: Create identities
```

To be consistent I would lowercase this. see above and below, we sometimes have it lowercased, sometimes uppercased as "proper term". 
But I think it would be less errorprone to lowercase everything that is not a super Ory specific term like Identity Schema.